source/quick-start.txt

```diff
@@ -119,6 +119,10 @@ includes information on the hostname or IP address and port of your
 instance, authentication mechanism, user credentials when applicable, and
 other connection options.
 
+If you are connecting to an instance or cluster that is not hosted by Atlas,
+see the :doc:`Connection Guide </fundamentals/connection>` for instructions on
```

Comment:
Suggestion: I think linking directly to the section may be easier for the user to find the localhost instructions.

Reply:
Thanks for the suggestion. Reformatted the connection page headers so that it redirects to a top-level header "other ways to connect to MongoDB".